Makefile

```diff
@@ -15,12 +15,16 @@ export TARGET
 help: ## Display this help text
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-install: node_modules ## Install dependencies locally
+install: ## Install dependencies locally
+	make --jobs=2 node_modules gitmodules
```

Comment:
Not familiar with recursive make, reading up on it.

Reply:
Thought: need to change what happens with `make dev` now.

Reply:
Consider `$(MAKE)`? https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html#MAKE-Variable

Reply:
From the little I know, the pattern should be to correctly specify dependencies and rely on a top level `-j N` for parallelization rather than having to use recursive make on the same Makefile of a small project; more investigation needed.

Reply:
Seems like passing along `MAKEFLAGS` _after_ setting `--jobs` is the way to go (so it overwrites it if being a sub make). https://github.com/libero/article-store/pull/34/commits/62e50d48a9c1408d3b9b764b636fa788d34c6454